src/app/story/directives/podcast.component.ts

```diff
@@ -33,12 +37,56 @@ export class PodcastComponent implements OnDestroy {
 
   story: StoryModel;
   tabSub: Subscription;
-  itunesExplicitDoc = 'https://help.apple.com/itc/podcasts_connect/#/itc1723472cb';
+  itunesRequirementsDoc = 'https://help.apple.com/itc/podcasts_connect/#/itc1723472cb';
+  itunesExplicitDoc = 'https://support.apple.com/en-us/HT202005';
 
-  OPTIONS = ['Explicit', 'Clean'];
+  OPTIONS = ['', 'Explicit', 'Clean'];
 
-  constructor(tab: TabService) {
-    this.tabSub = tab.model.subscribe((s: StoryModel) => this.story = s);
+  constructor(tab: TabService,
+              private cms: CmsService) {
+    this.tabSub = tab.model.subscribe((s: StoryModel) => this.init(s));
+  }
+
+  init(story) {
+    this.story = story;
+    this.getSeriesOverrides();
+  }
+
+  getSeriesOverrides() {
+    this.cms.account.subscribe(account => {
+      if (this.story.isNew && this.story.parent && this.story.parent.isa('series')) {
+        this.defaultDistributionOverrides(new SeriesModel(account, this.story.parent, false));
+      } else if (!this.story.isNew && this.story.doc.has('prx:series')) {
+        this.story.doc.follow('prx:series').subscribe(series => this.defaultDistributionOverrides(new SeriesModel(account, series, false)));
+      }
+    });
+  }
+
+  defaultDistributionOverrides(series: SeriesModel) {
+    if (series.doc && series.doc.count('prx:distributions')) {
+      series.doc.followItems('prx:distributions').subscribe(distributions => {
+        let models = distributions.map(d => new DistributionModel(series.doc, d, false));
+        this.defaultExplicitToSeries(models.concat(series.unsavedDistribution).filter(d => d));
+      });
+    } else if (series.unsavedDistribution) {
+      this.defaultExplicitToSeries([series.unsavedDistribution]);
+    }
+  }
+
+  defaultExplicitToSeries(distributions: DistributionModel[]) {
+    let dist = distributions.find(d => d.kind === 'podcast');
+    if (dist) {
+      dist.loadExternal().subscribe(() => {
+        // TODO: how to know which of these versions is the podcast?
+        this.story.versions.forEach(v => {
+          if (dist.podcast && v.explicit === null) {
+            // TODO: how to know that audio version explicit has not been set?
+            // audio version explicit is empty string when it has not been set, which is a valid value
+            v.explicit = dist.podcast.explicit;
```

Comment:
There's gonna be a useAsOriginal flag on BaseModel.set() soon, use that to override the initial empty string value instead of plain assignment. Having the override value as the original value allows us to use changed() to determine if the value should be overridden on draft stories.